README.md

```diff
@@ -23,6 +23,14 @@ Add following step to your workflow:
 
 Represents number of latest releases (sorted by `created_at`) to keep. Pass `0` if you want to delete all releases
 
+#### `delete_tags`
+
+| required | default |
+| -------- | ------- |
+| false    | false   |
+
+Represents whether to delete tags associated to older releases or not. Older tags without any associated releases will not be deleted
```

Comment:
```suggestion
Specifies whether to delete tags associated to older releases or not. Older tags without any associated releases will not be deleted
```

Reply:
Instead of "Represents....", you should probably use "Specifies..." for all those options in the README.md. Easier to translate for all languages. ;-)

Reply:
Thanks for suggestion. I will do that